frontend/templates/windowsTOS.html

```diff
@@ -0,0 +1,68 @@
+<!-- This is the html page for a user accepting the terms of service during the
+windwos mdm enrollment flow. -->
```

Comment:
Is there a reason to put this in the `frontend` directory rather than in `server`?